Classes/Loc+TripKit.swift

```diff
@@ -74,6 +78,18 @@ extension Loc {
       return String(format: format, NSNumber(value: count))
     }
   }
+  
+  public static func Alerts(_ count: Int) -> String {
+    switch count {
+    case 0: return NSLocalizedString("No alert", tableName: "TripKit", bundle: .tripKit, comment: "Number of alerts, in this case, there is none")
+      
+    case 1: return NSLocalizedString("1 alert", tableName: "TripKit", bundle: .tripKit, comment: "Number of alerts, in this case, there is only 1 (singular)")
+      
+    default:
```

Comment:
Do we really ever need the cases for 0 and 1? Could just return an optional string.